packages/e2e-tests/_test_dependencies/vite-plugins/index.js

```diff
@@ -1,5 +1,15 @@
 import path from 'node:path';
 import fs from 'node:fs';
+import MagicString from 'magic-string';
+
+function replaceWithSourceMap(code, value, replacement) {
```

Comment:
the missing sourcemap lead to error logs, add them to allow easier debugging and assertions that no unexpected errors happened